build/cpp/cmd/src/cmd.cc

```diff
@@ -11,12 +11,13 @@ cmd::CommandLine::~CommandLine() { delete func_client_; }
 
 // hooks the appropriate function ids and functions
 void cmd::CommandLine::Hook() {
-  func_client_->hook(0, "Registeruser");
-  func_client_->hook(1, "Warble");
-  func_client_->hook(2, "Follow");
-  func_client_->hook(3, "Read");
-  func_client_->hook(4, "Profile");
-  std::cout << "Warble functions successfully hooked." << std::endl;
+  func_client_->Hook(0, "Registeruser");
+  func_client_->Hook(1, "Warble");
+  func_client_->Hook(2, "Follow");
+  func_client_->Hook(3, "Read");
+  func_client_->Hook(4, "Profile");
+  func_client_->Hook(1, "Hashtags");
```

Comment:
One thing I noticed is you hook Hashtags with the value 1 along with Warble, but it looks like this was intentional.

Reply:
Thanks for pointing out. It should have been deleted. The stream map used in func is loaded in when the server starts